epilogue-processor/src/main/java/edu/wpi/first/epilogue/processor/ArrayHandler.java

```diff
@@ -67,6 +103,20 @@ public String logInvocation(Element element) {
           + ", "
           + m_structHandler.structAccess(componentType)
           + ")";
+    } else if (m_loggableHandler.isLoggableType(componentType)) {
+      var elementAccess = elementAccess(element);
+      var logInvocation =
+          m_loggableHandler.logInvocation(element, componentType, elementAccess + "[i]");
+      logInvocation =
+          logInvocation.replaceAll(
+              "backend\\.getNested\\(\".*\"\\)",
+              "backend.getNested(\"%s/\" + i)".formatted(loggedName(element)));
+      return """
+          for (int i = 0; i < %s.length; i++) {
```

Comment:
This fails for private arrays - it'll generate `(Foo[]) $array.get(object).length` instead of `((Foo[]) $array.get(object).length`

You'll need to wrap `%s` in parens to be safe

```suggestion
          for (int i = 0; i < (%s).length; i++) {
```